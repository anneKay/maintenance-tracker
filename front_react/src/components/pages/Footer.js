import React from 'react';
import footerOne from '../../img/contact-icon.png';
import footerTwo from '../../img/coffee.png';
import footerThree from '../../img/join.png';

const Footer = () => (
  <footer className="footer-container">
    <section>
      <div className="row">
        <div className="col-1-3 footer-content remove-gutter-xs">
          <img className="footer-img" src={footerOne} />
          <a>
            <h3>
                        Contact Us
            </h3>
          </a>
        </div>
        <div className="col-1-3 footer-content remove-gutter-xs">

          <img className="footer-img " src={footerTwo} />
          <a>
            <h3>
                        Read More
            </h3>
          </a>
        </div>
        <div className="col-1-3 footer-content remove-gutter-xs">

          <img className="footer-img " src={footerThree} />
          <a><h3 className="">Join Us</h3></a>
        </div>
      </div>
    </section>

    <section className="copyright">
      <div>
        <small>&copy; Copyright 2018, Maintenance Tracker</small>
      </div>
    </section>
  </footer>

);
export default Footer;
