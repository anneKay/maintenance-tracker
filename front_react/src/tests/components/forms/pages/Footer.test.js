import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../../../components/pages/Footer';


describe(' Footer Form component rendering', () => {
  const wrapper = shallow(<Footer />);

  it('Should render properly', () => {
    expect(wrapper.length).toBe(1);
    expect(wrapper).toMatchSnapshot();
  });
});
