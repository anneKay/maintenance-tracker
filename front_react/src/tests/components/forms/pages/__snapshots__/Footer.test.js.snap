// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Footer Form component rendering Should render properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <section>
          <div
            className="row"
          >
            <div
              className="col-1-3 footer-content remove-gutter-xs"
            >
              <img
                alt=""
                className="footer-img"
                src="test-file-stub"
              />
              <Link
                replace={false}
                to="/"
              >
                <h3>
                  Contact Us
                </h3>
              </Link>
            </div>
            <div
              className="col-1-3 footer-content remove-gutter-xs"
            >
              <img
                alt=""
                className="footer-img "
                src="test-file-stub"
              />
              <Link
                replace={false}
                to="/"
              >
                <h3>
                  Read More 
                </h3>
              </Link>
            </div>
            <div
              className="col-1-3 footer-content remove-gutter-xs"
            >
              <img
                alt="/"
                className="footer-img "
                src="test-file-stub"
              />
              <Link
                replace={false}
                to="/signup"
              >
                <h3
                  className=""
                >
                  Join Us
                </h3>
              </Link>
            </div>
          </div>
        </section>,
        <section
          className="copyright"
        >
          <div>
            <small>
              Â© Copyright 2018, Maintenance Tracker
            </small>
          </div>
        </section>,
      ],
      "className": "footer-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="row"
          >
            <div
              className="col-1-3 footer-content remove-gutter-xs"
            >
              <img
                alt=""
                className="footer-img"
                src="test-file-stub"
              />
              <Link
                replace={false}
                to="/"
              >
                <h3>
                  Contact Us
                </h3>
              </Link>
            </div>
            <div
              className="col-1-3 footer-content remove-gutter-xs"
            >
              <img
                alt=""
                className="footer-img "
                src="test-file-stub"
              />
              <Link
                replace={false}
                to="/"
              >
                <h3>
                  Read More 
                </h3>
              </Link>
            </div>
            <div
              className="col-1-3 footer-content remove-gutter-xs"
            >
              <img
                alt="/"
                className="footer-img "
                src="test-file-stub"
              />
              <Link
                replace={false}
                to="/signup"
              >
                <h3
                  className=""
                >
                  Join Us
                </h3>
              </Link>
            </div>
          </div>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="col-1-3 footer-content remove-gutter-xs"
              >
                <img
                  alt=""
                  className="footer-img"
                  src="test-file-stub"
                />
                <Link
                  replace={false}
                  to="/"
                >
                  <h3>
                    Contact Us
                  </h3>
                </Link>
              </div>,
              <div
                className="col-1-3 footer-content remove-gutter-xs"
              >
                <img
                  alt=""
                  className="footer-img "
                  src="test-file-stub"
                />
                <Link
                  replace={false}
                  to="/"
                >
                  <h3>
                    Read More 
                  </h3>
                </Link>
              </div>,
              <div
                className="col-1-3 footer-content remove-gutter-xs"
              >
                <img
                  alt="/"
                  className="footer-img "
                  src="test-file-stub"
                />
                <Link
                  replace={false}
                  to="/signup"
                >
                  <h3
                    className=""
                  >
                    Join Us
                  </h3>
                </Link>
              </div>,
            ],
            "className": "row",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt=""
                    className="footer-img"
                    src="test-file-stub"
                  />,
                  <Link
                    replace={false}
                    to="/"
                  >
                    <h3>
                      Contact Us
                    </h3>
                  </Link>,
                ],
                "className": "col-1-3 footer-content remove-gutter-xs",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "footer-img",
                    "src": "test-file-stub",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <h3>
                      Contact Us
                    </h3>,
                    "replace": false,
                    "to": "/",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Contact Us",
                    },
                    "ref": null,
                    "rendered": "Contact Us",
                    "type": "h3",
                  },
                  "type": [Function],
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt=""
                    className="footer-img "
                    src="test-file-stub"
                  />,
                  <Link
                    replace={false}
                    to="/"
                  >
                    <h3>
                      Read More 
                    </h3>
                  </Link>,
                ],
                "className": "col-1-3 footer-content remove-gutter-xs",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "",
                    "className": "footer-img ",
                    "src": "test-file-stub",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <h3>
                      Read More 
                    </h3>,
                    "replace": false,
                    "to": "/",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Read More ",
                    },
                    "ref": null,
                    "rendered": "Read More ",
                    "type": "h3",
                  },
                  "type": [Function],
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <img
                    alt="/"
                    className="footer-img "
                    src="test-file-stub"
                  />,
                  <Link
                    replace={false}
                    to="/signup"
                  >
                    <h3
                      className=""
                    >
                      Join Us
                    </h3>
                  </Link>,
                ],
                "className": "col-1-3 footer-content remove-gutter-xs",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "/",
                    "className": "footer-img ",
                    "src": "test-file-stub",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <h3
                      className=""
                    >
                      Join Us
                    </h3>,
                    "replace": false,
                    "to": "/signup",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Join Us",
                      "className": "",
                    },
                    "ref": null,
                    "rendered": "Join Us",
                    "type": "h3",
                  },
                  "type": [Function],
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "section",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div>
            <small>
              Â© Copyright 2018, Maintenance Tracker
            </small>
          </div>,
          "className": "copyright",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <small>
              Â© Copyright 2018, Maintenance Tracker
            </small>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Â© Copyright 2018, Maintenance Tracker",
            },
            "ref": null,
            "rendered": "Â© Copyright 2018, Maintenance Tracker",
            "type": "small",
          },
          "type": "div",
        },
        "type": "section",
      },
    ],
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <section>
            <div
              className="row"
            >
              <div
                className="col-1-3 footer-content remove-gutter-xs"
              >
                <img
                  alt=""
                  className="footer-img"
                  src="test-file-stub"
                />
                <Link
                  replace={false}
                  to="/"
                >
                  <h3>
                    Contact Us
                  </h3>
                </Link>
              </div>
              <div
                className="col-1-3 footer-content remove-gutter-xs"
              >
                <img
                  alt=""
                  className="footer-img "
                  src="test-file-stub"
                />
                <Link
                  replace={false}
                  to="/"
                >
                  <h3>
                    Read More 
                  </h3>
                </Link>
              </div>
              <div
                className="col-1-3 footer-content remove-gutter-xs"
              >
                <img
                  alt="/"
                  className="footer-img "
                  src="test-file-stub"
                />
                <Link
                  replace={false}
                  to="/signup"
                >
                  <h3
                    className=""
                  >
                    Join Us
                  </h3>
                </Link>
              </div>
            </div>
          </section>,
          <section
            className="copyright"
          >
            <div>
              <small>
                Â© Copyright 2018, Maintenance Tracker
              </small>
            </div>
          </section>,
        ],
        "className": "footer-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="row"
            >
              <div
                className="col-1-3 footer-content remove-gutter-xs"
              >
                <img
                  alt=""
                  className="footer-img"
                  src="test-file-stub"
                />
                <Link
                  replace={false}
                  to="/"
                >
                  <h3>
                    Contact Us
                  </h3>
                </Link>
              </div>
              <div
                className="col-1-3 footer-content remove-gutter-xs"
              >
                <img
                  alt=""
                  className="footer-img "
                  src="test-file-stub"
                />
                <Link
                  replace={false}
                  to="/"
                >
                  <h3>
                    Read More 
                  </h3>
                </Link>
              </div>
              <div
                className="col-1-3 footer-content remove-gutter-xs"
              >
                <img
                  alt="/"
                  className="footer-img "
                  src="test-file-stub"
                />
                <Link
                  replace={false}
                  to="/signup"
                >
                  <h3
                    className=""
                  >
                    Join Us
                  </h3>
                </Link>
              </div>
            </div>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="col-1-3 footer-content remove-gutter-xs"
                >
                  <img
                    alt=""
                    className="footer-img"
                    src="test-file-stub"
                  />
                  <Link
                    replace={false}
                    to="/"
                  >
                    <h3>
                      Contact Us
                    </h3>
                  </Link>
                </div>,
                <div
                  className="col-1-3 footer-content remove-gutter-xs"
                >
                  <img
                    alt=""
                    className="footer-img "
                    src="test-file-stub"
                  />
                  <Link
                    replace={false}
                    to="/"
                  >
                    <h3>
                      Read More 
                    </h3>
                  </Link>
                </div>,
                <div
                  className="col-1-3 footer-content remove-gutter-xs"
                >
                  <img
                    alt="/"
                    className="footer-img "
                    src="test-file-stub"
                  />
                  <Link
                    replace={false}
                    to="/signup"
                  >
                    <h3
                      className=""
                    >
                      Join Us
                    </h3>
                  </Link>
                </div>,
              ],
              "className": "row",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <img
                      alt=""
                      className="footer-img"
                      src="test-file-stub"
                    />,
                    <Link
                      replace={false}
                      to="/"
                    >
                      <h3>
                        Contact Us
                      </h3>
                    </Link>,
                  ],
                  "className": "col-1-3 footer-content remove-gutter-xs",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "",
                      "className": "footer-img",
                      "src": "test-file-stub",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": <h3>
                        Contact Us
                      </h3>,
                      "replace": false,
                      "to": "/",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Contact Us",
                      },
                      "ref": null,
                      "rendered": "Contact Us",
                      "type": "h3",
                    },
                    "type": [Function],
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <img
                      alt=""
                      className="footer-img "
                      src="test-file-stub"
                    />,
                    <Link
                      replace={false}
                      to="/"
                    >
                      <h3>
                        Read More 
                      </h3>
                    </Link>,
                  ],
                  "className": "col-1-3 footer-content remove-gutter-xs",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "",
                      "className": "footer-img ",
                      "src": "test-file-stub",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": <h3>
                        Read More 
                      </h3>,
                      "replace": false,
                      "to": "/",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Read More ",
                      },
                      "ref": null,
                      "rendered": "Read More ",
                      "type": "h3",
                    },
                    "type": [Function],
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <img
                      alt="/"
                      className="footer-img "
                      src="test-file-stub"
                    />,
                    <Link
                      replace={false}
                      to="/signup"
                    >
                      <h3
                        className=""
                      >
                        Join Us
                      </h3>
                    </Link>,
                  ],
                  "className": "col-1-3 footer-content remove-gutter-xs",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "/",
                      "className": "footer-img ",
                      "src": "test-file-stub",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": <h3
                        className=""
                      >
                        Join Us
                      </h3>,
                      "replace": false,
                      "to": "/signup",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Join Us",
                        "className": "",
                      },
                      "ref": null,
                      "rendered": "Join Us",
                      "type": "h3",
                    },
                    "type": [Function],
                  },
                ],
                "type": "div",
              },
            ],
            "type": "div",
          },
          "type": "section",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div>
              <small>
                Â© Copyright 2018, Maintenance Tracker
              </small>
            </div>,
            "className": "copyright",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <small>
                Â© Copyright 2018, Maintenance Tracker
              </small>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Â© Copyright 2018, Maintenance Tracker",
              },
              "ref": null,
              "rendered": "Â© Copyright 2018, Maintenance Tracker",
              "type": "small",
            },
            "type": "div",
          },
          "type": "section",
        },
      ],
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
